Code/Tools/ProjectManager/Source/FormLineEditTagsWidget.cpp

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <FormLineEditTagsWidget.h>
+#include <AzQtComponents/Components/StyledLineEdit.h>
+#include <FormLineEditWidget.h>
+#include <AzQtComponents/Components/Widgets/LineEdit.h>
+#include <QPushButton>
+#include <QAbstractItemView>
+#include <QSpacerItem>
+#include <QCompleter>
+#include <QFile>
+#include <QVBoxLayout>
+#include <QHBoxLayout>
+#include <QCheckBox>
+#include <QLineEdit>
+#include <QLabel>
+#include <QMovie>
+#include <QFrame>
+#include <QValidator>
+#include <QStyle>
+#include <QKeyEvent>
+
+namespace O3DE::ProjectManager
+{
+
+    void FormLineEditTagsWidget::setupCompletionTags()
+    {
+        m_completionTags << "Audio" << "Animation" << "Physics" << "GameDevelopment" << "IK" << "Blender" << "Rendering" << "Terrain";
```

Comment:
What was the reasoning behind these choices? IK seems very specific when tags that are used a lot (Framework, Scripting,  Tools, Utility) are not in this list.

Reply:
I find it strange that is hardcoded and not data driven from a file.

Reply:
Making it data driven was intended to be the next step after implementing the tags field. I went with a preliminary set of tags primarily to test the auto-completion features. Most of the effort for this PR went into fixing the interactions and styling of the completer.

As for tag choice, these were the tags I was most aware of at the time, but I can add more tags in. 

Regarding the data file, should I add a text file in the existing resources folder (i.e. where we keep the qss stylesheets)? I could call it something like `completionTags.txt`

Reply:
Next commit will have this part updated to a data driven format. I opted to use tags commonly recognized in the existing gem catalog.